src/main/java/edu/usf/cutr/gtfsrtvalidator/background/BackgroundTask.java

```diff
@@ -180,16 +180,16 @@ public void run() {
 
             for (GtfsRtFeedModel gtfsRtFeedModel : gtfsRtFeedModelList) {
                 GtfsRealtime.FeedMessage message = mGtfsRtFeedMap.get(gtfsRtFeedModel.getGtfsRtId());
-                if (header == null) {
-                    // Save one header to use in our combined feed below
-                    header = message.getHeader();
-                } else {
-                    if (message.getHeader() != null && message.getHeader().getTimestamp() > header.getTimestamp()) {
-                        // Use largest header timestamp with multiple feeds - see #239
+                if (message != null) {
```

Comment:
@Suryakandukoori I don't think this section of the changes should be in this PR.  Looks like this might be a fix for https://github.com/CUTR-at-USF/gtfs-realtime-validator/issues/253?  Could you please remove this section of changes from this PR?